docs/index.asciidoc

```diff
@@ -109,14 +111,80 @@ append and indirect cannot be combined and will fail validation. +
 `%{&+something}` will add a value to the `+something` key, again probably unintended. +
 ===============================
 
-*Delimiter repetition:* +
-In the source text if a field has variable width padded with delimiters, the padding will be ignored. +
-e.g. for texts of:
+*Behaviour when multiple delimiter are found in the source text:* +
+
+[IMPORTANT]
+===============================
+Starting from version 1.1.1 of this plugin, multiple found delimiter handling has changed.
+Now multiple delimters will be seen as missing fields by default and not padding.
+If you are already using Dissect and your source text has fields padded with extra delimiters,
+you will need to change your config. Please read the section below.
+===============================
+
+**Empty data between delimiters** +
+Given this text as the sample used to create a dissection:
+....
+John Smith,Big Oaks,Wood Lane,Hambledown,Canterbury,CB34RT
+....
+The created dissection, with 6 fields, is:
+....
+%{name},%{addr1},%{addr2},%{addr3},%{city},%{zip}
+....
+When a line like this is processed:
+....
+Jane Doe,4321 Fifth Avenue,,,New York,87432
+....
+Dissect will create an event with empty fields for `addr2 and addr3` like so:
+....
+{
+  "name": "Jane Doe",
+  "addr1": "4321 Fifth Avenue",
+  "addr2": "",
+  "addr3": "",
+  "city": "New York"
+  "zip": "87432"
+}
+....
+
+**Delimiters used as padding to visually align fields** +
+Padding to the right hand side +
+Given these texts as the samples used to create a dissection:
+....
+00000043 ViewReceive     machine-321
+f3000a3b Calc            machine-123
+....
+The dissection, with 3 fields, is:
+....
+%{id} %{function->} %{server}
+....
+Note, above, the second field has a `->` suffix, this tells Dissect to ignore padding to its right. +
```

Comment:
comma splice. Change this to say "...suffix, which tells" or "...suffix. This tells"

Reply:
again I see a trailing plus sign and I'm not sure what it's meant to do.